sql-imap-test/src/main/java/com/hazelcast/jet/tests/sql/tests/SqlObjectMapTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.sql.tests;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.JetInstance;
+
+//TODO: Create SQL tests for different InMemoryFormat
+//TODO: Create SQL tests for Kafka as a data source
+public class SqlObjectMapTest extends AbstractSqlTest {
+
+    private static final String MAP_NAME = "object_sql_map";
+
+    private HazelcastInstance hazelcastInstance;
```

Comment:
Why?

Reply:
It's not an issue but I don't like it since it bypasses the structure from `AbstractSoakTest`. On the other hand I don't have a technical argument for not to have it like this since it's just a holder => feel free to keep it as it is.

Btw. should not it be init in `AbstractSqlMapTest` instead of classes which extends it?

Reply:
Ok, I changed that to JetInstance. I was passing it to one of the methods within AbstractSqlMapTest, so maybe it does make sense to make it work with JetInstance rather than HazelcastInstance. Thanks!